api/import_api.go

```diff
@@ -0,0 +1,91 @@
+package api
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/ONSdigital/go-ns/log"
+	"net/http"
+	"net/url"
+)
+
+// ImportAPI aggregates a client and URL and other common data for accessing the API
+type ImportAPI struct {
+	Client    *http.Client
+	URL       string
+	AuthToken string
+}
+
+// ImportJob comes from the Import API and links an import job to its (other) instances
+type ImportJob struct {
+	JobID     string         `json:"job_id"`
+	Instances []InstanceLink `json:"instances"`
+}
+
+// InstanceLink identifies an (instance or import-job) by id and URL (from Import API)
+type InstanceLink struct {
+	ID   string `json:"id"`
+	Link string `json:"link"`
+}
+
+// NewImportAPI creates an ImportAPI object
+func NewImportAPI(client *http.Client, url, authToken string) *ImportAPI {
```

Comment:
Constructor not needed?

Reply:
The vars should not have been public. I have made them private.